src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -2,7 +2,7 @@ class Webui::Users::NotificationsController < Webui::WebuiController
   include Webui::NotificationsFilter
 
   ALLOWED_FILTERS = %w[all comments requests incoming_requests outgoing_requests relationships_created relationships_deleted build_failures
-                       reports reviews workflow_runs appealed_decisions].freeze
+                       reports reviews workflow_runs appealed_decisions added_user_to_groups removed_user_from_groups user_on_groups].freeze
```

Comment:
Same here, there is on `user_on_groups` left.

Reply:
Done